salt/platform-testing/general.sls

```diff
@@ -43,6 +44,14 @@
   {%- endfor -%}
 {%- endif -%}
 
+{%- set fh_hosts = [] -%}
+{%- set fh_nodes = salt['pnda.ip_addresses']('flink') -%}
```

Comment:
we are not relying anymore on ip_addresses now that we are putting Consul so can you put pnda.get_hosts_for_role instead please?